modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -2326,18 +2325,49 @@ protected GridCoverage2D readCoverage(
                                 GridGeometry2D readGG)
                                 throws IOException {
                             Interpolation interpolation = getRenderingInterpolation(layer);
-                            GridCoverageReaderHelper helper;
+                            RenderingHints interpolationHints =
+                                    new RenderingHints(JAI.KEY_INTERPOLATION, interpolation);
+                            final GridCoverageRenderer gcr;
                             try {
-                                helper =
-                                        new GridCoverageReaderHelper(
+                                // Use the original screenSize
+                                // the GridCoverageRenderer will take care of eventual gutter or
```

Comment:
Is this actually true if the advanced projection handling is off though?

Reply:
I have found another place where it's raising issues... backing to the original gridgeometry usage